core/domain/interaction_validation_jobs_one_off.py

```diff
@@ -0,0 +1,278 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""One-off jobs for interaction validation."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import ast
+import itertools
+import re
+
+from core import jobs
+from core.domain import customization_args_util
+from core.domain import exp_fetchers
+from core.domain import html_validation_service
+from core.domain import interaction_registry
+from core.domain import rights_manager
+from core.platform import models
+import feconf
+import python_utils
+
+(exp_models,) = models.Registry.import_models([
+    models.NAMES.exploration])
+
+ADDED_THREE_VERSIONS_TO_GCS = 'Added the three versions'
+_COMMIT_TYPE_REVERT = 'revert'
+ALL_IMAGES_VERIFIED = 'Images verified'
+ERROR_IN_FILENAME = 'There is an error in the filename'
+FILE_COPIED = 'File Copied'
+FILE_ALREADY_EXISTS = 'File already exists in GCS'
+FILE_FOUND_IN_GCS = 'File found in GCS'
```

Comment:
Can we remove unwanted constants from here?

Reply:
Done - none of them being are used. I was expecting the linter to flag this.
\cc @bansalnitish can we have this as a lint test if it isn't present already?